packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -143,14 +153,40 @@ class ChatMessage extends UIComponent<ReactProps<ChatMessageProps>, ChatMessageS
     _.invoke(this.props, 'onBlur', e, this.props)
   }
 
+  renderReactions = (styles: ComponentSlotStylesPrepared) => {
+    const { reactions } = this.props
+    return (
+      <Box styles={styles.reactions}>
+        {_.map(reactions, reaction => {
+          return Reaction.create(reaction, {
+            defaultProps: {
+              styles: styles.reaction,
+              as: 'span',
+            },
+          })
+        })}
+      </Box>
+    )
+  }
```

Comment:
Should we introduce a separate component `ReactionGroup`?

```jsx
<ReactionGroup items={[]} />
```

```jsx
ReactionGroup.create(reactions)
```

Reply:
Added.